src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,5 @@
+$pageFont: Roboto, sans-serif;
+$contentWidth: 166px;
```

Comment:
```suggestion
$contentWidth: 100%;
```

Reply:
When I change the value to 100%, the entire text is stretched by 200px, while the image tag has 160px and is centered

How can I better add indents to the text from the article, if the padding for the article does not reduce the blocks of text and the size of these blocks is inherited at 200px?

Thank you in advance